backend/databasemodel/model.sql

```diff
@@ -1308,39 +1311,38 @@ CREATE TABLE kooste.lipas_pisteet (
 	"phoneNumber" text,
 	www text,
 	season text NOT NULL,
-	toilet boolean DEFAULT NULL,
-	shower boolean DEFAULT NULL,
-	"changingRooms" boolean DEFAULT NULL,
-	ligthing boolean DEFAULT NULL,
-	"restPlacesCount" numeric DEFAULT NULL,
-	"skiTrackTraditional" boolean DEFAULT NULL,
-	"skiTrackFreestyle" boolean DEFAULT NULL,
-	"litRouteLengthKm" numeric DEFAULT NULL,
-	"routeLengthKm" numeric DEFAULT NULL,
-	pier boolean DEFAULT NULL,
-	"otherPlatforms" boolean DEFAULT NULL,
-	"accessibilityInfo" text DEFAULT NULL,
-	kiosk boolean DEFAULT NULL,
-	"skiService" boolean DEFAULT NULL,
-	"equipmentRental" boolean DEFAULT NULL,
-	sauna boolean DEFAULT NULL,
-	playground boolean DEFAULT NULL,
-	"exerciseMachines" boolean DEFAULT NULL,
-	"infoFi" text DEFAULT NULL,
-	"trackLengthM" numeric DEFAULT NULL,
-	"altitudeDifference" numeric DEFAULT NULL,
-	"climbingWallWidthM" numeric DEFAULT NULL,
-	"climbingWallHeightM" numeric DEFAULT NULL,
-	"climbingRoutesCount" numeric DEFAULT NULL,
-	"holesCount" numeric DEFAULT NULL,
-	"boatLaunchingSpot" boolean DEFAULT NULL,
-	"iceClimbing" boolean DEFAULT NULL,
-	"iceReduction" boolean DEFAULT NULL,
-	range boolean DEFAULT NULL,
-	"trackType" text DEFAULT NULL,
-	status text DEFAULT NULL,
```

Comment:
Should we add `visibility` field if we remove `status`?

Reply:
This is something we discussed via Slack. We decided (with Sanna) to leave visibility field out from lipas tables since we want to show each feature in lipas belonging to the predefined type codes. If there is something they do not want to show, lipas will be used for deleting such objects.